### PR TITLE
Minor Intent to import description fix

### PR DESCRIPTION
**Pull request by nissimsan**

### CODE REVIEW

**Comment by nissimsan** [comment]
This will have a merge conflict

**Review by mprorock — APPROVED** [review]

File: docs/openapi/components/schemas/common/IntentToImport.yml [hunk 1]
```diff
@@ -5,11 +5,10 @@ title: Intent To Import
...
   A simple case where an entity (person or organization) wishes to notify some
   third party that they intend to import some product, likely into a different
-  geographic region than that of its origin. This is a highly simplified version
-  of an LOI that can be used to identify and declare intent earlier in the
-  process than would be possible otherwise. This object may be easily
-  subclassed for extension with other data for other cases where additional
-  regulatory data may be required.
+  geographic region than that of its origin. This can be used to identify and 
+  declare intent earlier in the process than would be possible otherwise. This 
+  object may be easily subclassed for extension with other data for other cases
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/IntentToImport.yml:10]:
```suggestion
  object may be easily subclassed for extension with other data for cases 
```

File: docs/openapi/components/schemas/credentials/IntentToImportCertificate.yml [hunk 2]
```diff
@@ -5,11 +5,10 @@ title: Intent To Import Certificate
...
   A simple case where an entity (person or organization) wishes to notify some
   third party that they intend to import some product, likely into a different
-  geographic region than that of its origin. This is a highly simplified version
-  of an LOI that can be used to identify and declare intent earlier in the
-  process than would be possible otherwise. This object may be easily
-  subclassed for extension with other data for other cases where additional
-  regulatory data may be required.
+  geographic region than that of its origin. This can be used to identify and 
+  declare intent earlier in the process than would be possible otherwise. This 
+  object may be easily subclassed for extension with other data for other cases
```
Comment (TallTed) [inline docs/openapi/components/schemas/credentials/IntentToImportCertificate.yml:10]:
```suggestion
  object may be easily subclassed for extension with other data for cases 
```

**Review by BenjaminMoe — APPROVED** [review]

**Review by mkhraisha — APPROVED** [review]

**Comment by nissimsan** [comment]
Merging offline pending conflict resolve.

**Comment by nissimsan** [comment]
Merge conflict resolved.

**Pull request merged**